docs/concepts/scopes.md

```diff
@@ -10,6 +10,9 @@ evaluation preceding allow.
 
 Scopes are a broader way of allowing or blocking `execve()`s.
```

Comment:
Please change this to:
`Scopes are a broader way of allowing or blocking executions`.